app/assets/stylesheets/provider/_buttons.scss

```diff
@@ -74,13 +74,6 @@ a.next {
   @include internal-potato-button(white, $delete-color, $delete-color );
 }
 
-
-.formtastic .actions {
```

Comment:
To be honest, I am not 100% sure this does not have any negative effect on any existing forms... But I think there are very few forms left that would use it.